TShockAPI/TShockAPI.csproj

```
@@ -1,242 +1,58 @@
-    <Compile Include="Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Include="Rest\Rest.cs" />
-    <Compile Include="Rest\RestCommand.cs" />
-    <Compile Include="Rest\RestManager.cs" />
-    <Compile Include="Rest\RestObject.cs" />
-    <Compile Include="Rest\RestVerbs.cs" />
-    <Compile Include="Rest\SecureRest.cs" />
-    <Compile Include="Configuration\ServerSideConfig.cs" />
-    <Compile Include="TSServerPlayer.cs" />
-    <Compile Include="Utils.cs" />
-    <Compile Include="TShock.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="TSPlayer.cs" />
-    <Compile Include="UpdateManager.cs" />
-    <Compile Include="DB\WarpsManager.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="packages.config" />
-    <None Include="TShockAPI.licenseheader" />
+    <PackageReference Include="BCrypt.Net-Next" Version="4.0.3" />
+    <PackageReference Include="MySql.Data" Version="8.0.30" />
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="6.0.8" />
   </ItemGroup>
+
   <ItemGroup>
-    <EmbeddedResource Include="Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <SubType>Designer</SubType>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+    <ProjectReference Include="..\TerrariaServerAPI\TerrariaServerAPI\TerrariaServerAPI.csproj" />
   </ItemGroup>
+
   <ItemGroup>
-    <BootstrapperPackage Include=".NETFramework,Version=v4.0">
-      <Visible>False</Visible>
-      <ProductName>Microsoft .NET Framework 4 %28x86 and x64%29</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Client.3.5">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1 Client Profile</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Windows.Installer.3.1">
-      <Visible>False</Visible>
-      <ProductName>Windows Installer 3.1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
+    <Reference Include="HttpServer">
+      <HintPath>..\prebuilts\HttpServer.dll</HintPath>
+    </Reference>
   </ItemGroup>
+
   <ItemGroup>
-    <ProjectReference Include="..\TerrariaServerAPI\TerrariaServerAPI\TerrariaServerAPI.csproj">
-      <Project>{6877506e-adc6-4142-98a6-79e4fa02855a}</Project>
-      <Name>TerrariaServerAPI</Name>
-    </ProjectReference>
+    <None Include="..\prebuilts\HttpServer.dll">
+      <Link>HttpServer.dll</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

```suggestion
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```
Laxes requirements for incremental builds, or allows Visual Studio to skip the project more